build.py

```diff
@@ -5,16 +5,17 @@
 from jinja2 import Environment, FileSystemLoader
 
 
-env = Environment(loader=FileSystemLoader(["pages", "templates"]), autoescape=True)
+env = Environment(loader=FileSystemLoader(
+    ["pages", "templates"]), autoescape=True)
```

Comment:
right here is where we tell it to look in both the `pages` and `templates` directories in the working dir (repo root)